source/whats-new.txt

```diff
@@ -41,6 +41,18 @@ What's New in 5.0
 
 New features of the 5.0 {+driver-short+} release include:
 
+.. important:: Breaking Change: Removal of Callback Support
+
+   This release removes support for callbacks in favor of a promise-based API.
+   The following list provides some strategies for callback users to adopt this
+   version: 
+
+   - Migrate to the promise-based API (recommended)
+   - Use the promise-based API and ``util.callbackify``
+   - Add ``mongodb-legacy`` to continue using callbacks
+
+   For more information about these strategies, see the `v5 Change Notes <https://github.com/mongodb/node-mongodb-native/blob/main/etc/notes/CHANGES_5.0.0.md>`__.
```

Comment:
[question] I know this does not have to do with these docs obviously, but I wanted to point out that this link points to an entry that still has "TODO" under the TOC, which seems less-than-ideal. So I don't know if you want to pass that up the chain.

This in no way inhibits this pull request though.

Reply:
Good catch! I'll be double checking this when the version drops and will change the link or escalate the issue depending on which is needed